vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+#undef WINRT_LEAN_AND_MEAN
```

Comment:
this _feels_ like it's working around a more fundamental problem?

Reply:
According to the C++/WinRT team, this is the only mechanism that allows to extend an otherwise exclusive interface.
Because of WinInet internals, we had to write our own HTTP redirect logic, and wrap the existing `IHttpBaseProtocolFilter` implementation, while retaining the interface's `AllowAutoRedirect` property.

See `RedirectHttpFilter::RedirectHttpFilter` and `ResponseOperation OriginPolicyHttpFilter::SendRequestAsync` (~ line 751).